### ISSUE TITLE
Add code

### ISSUE THREAD

**Issue by Egor179** [OCB7D2D/BugfxFlickerTAA#2]
Can you adapt some more of the code?
There I also set the values to 128
![Screenshot_29](https://user-images.githubusercontent.com/18228429/147088531-af1ec0f0-2281-42b3-9abc-a2c6819e1b91.png)
![Screenshot_30](https://user-images.githubusercontent.com/18228429/147088534-db59e693-563e-46a8-bc59-162e7cb4d97a.png)

**Comment by Egor179** [comment]
And also this
This includes forced anisotropic filtering
![image](https://user-images.githubusercontent.com/18228429/147089429-eefed45c-5760-4e28-b81c-28bd20f17847.png)

**Comment by mgreter** [comment]
Can you create a Pull Request?
Would really be easier than with images ;)

**Comment by Egor179** [comment]
using System;
using System.Collections;
using UnityEngine;

// Token: 0x020008A0 RID: 2208
public static partial class GameOptionsManager
{
	// Token: 0x06005F8F RID: 24463
	public static void ApplyTextureFilter()
	{
		int textureFilter = GameOptionsManager.GetTextureFilter();
		QualitySettings.anisotropicFiltering = ((textureFilter == 0) ? AnisotropicFiltering.ForceEnable : ((textureFilter <= 3) ? AnisotropicFiltering.ForceEnable : AnisotropicFiltering.ForceEnable));
		if (GameOptionsManager.TextureFilterChanged != null)
		{
			GameOptionsManager.TextureFilterChanged(textureFilter);
		}
		Log.Out("ApplyTextureFilter {0}, AF {1}", new object[]
		{
			textureFilter,
			QualitySettings.anisotropicFiltering
		});
	}
}

**Comment by Egor179** [comment]
`using System;
using System.Collections;
using UnityEngine;

// Token: 0x020008A0 RID: 2208
public static partial class GameOptionsManager
{
	// Token: 0x06005F8F RID: 24463
	public static void ApplyTextureFilter()
	{
		int textureFilter = GameOptionsManager.GetTextureFilter();
		QualitySettings.anisotropicFiltering = ((textureFilter == 0) ? AnisotropicFiltering.ForceEnable : ((textureFilter <= 3) ? AnisotropicFiltering.ForceEnable : AnisotropicFiltering.ForceEnable));
		if (GameOptionsManager.TextureFilterChanged != null)
		{
			GameOptionsManager.TextureFilterChanged(textureFilter);
		}
		Log.Out("ApplyTextureFilter {0}, AF {1}", new object[]
		{
			textureFilter,
			QualitySettings.anisotropicFiltering
		});
	}
}
`

**Comment by Egor179** [comment]
> Can you create a Pull Request? Would really be easier than with images ;)

What is it like?

**Comment by mgreter** [comment]
You've never made a PR? No problem.
Paste the code again here (as you did).
I'll figure something out.

**Comment by Egor179** [comment]
https://github.com/Egor179/r/blob/main/code.cs

**Comment by mgreter** [comment]
OK, give me a minute. I'll try to come up with a Harmony PR.

**Comment by Egor179** [comment]
> You've never made a PR? No problem. Paste the code again here (as you did). I'll figure something out.

No, I don’t understand programming, I’m just looking for intuitive code and editing at random

**Comment by Egor179** [comment]
In fact, you only need these parts of the cod
![Screenshot_32](https://user-images.githubusercontent.com/18228429/147106107-6d356c29-b43c-445a-867e-be2c9530a4df.png)
![Screenshot_34](https://user-images.githubusercontent.com/18228429/147106110-20220f72-46b5-459e-9c21-dd79be02bd3b.png)
![Screenshot_33](https://user-images.githubusercontent.com/18228429/147106111-283911d7-08e5-47a9-9ccc-5d8faf17695d.png)
e

**Comment by mgreter** [comment]
See #3 for a PR (you can checkout the branch to test it).
I left out the changes for `ResetGraphicsOptions()`, I don't think these make much sense and they are only called when the game is reset, which doesn't seem to happen if you just configure the options. Not sure exactly how to trigger. Also the code in `ApplyTextureFilter` should work correctly if you set the setting accordingly. Or is there any reason to have `ForceEnable` when `OptionsGfxTexFilter` is set lower than `4`?

**Comment by Egor179** [comment]
> 

I do not know what the reason is, but at maximum settings, anisotropic filtering still does not work, until then I will not turn ForceEnable

**Comment by Egor179** [comment]
https://youtu.be/TviEblK1h1Q

**Comment by Egor179** [comment]
> https://youtu.be/TviEblK1h1Q

Did you see?
